### PR TITLE
adding scripts to generate society, democracy and tech-comm accounts

### PR DESCRIPTION
**Pull request by karthikeyan-cere**
Added script to generate accounts for democracy, society, tech-comm
Updated package.json
Added env to have the number of accounts for genesis validators, validators, democracy, society, tech-comm.

### CODE REVIEW

File: scripts/generate-accounts/index.ts [hunk 1]
```diff
@@ -23,12 +24,21 @@ class Accounts {
...
 
-    const validatorGenesisAccounts = await this.generateGenesisValidatorsAccounts(2);
+    const validatorGenesisAccounts = await this.generateGenesisValidatorsAccounts();
     console.log('\n');
 
-    const validatorAccounts = await this.generateValidatorsAccounts(3);
+    const validatorAccounts = await this.generateValidatorsAccounts();
     console.log('\n');
 
+    const generateDemocracyAccount = await this.generateDemocracyAccount();
```
Comment (AndreiNavoichyk) [inline scripts/generate-accounts/index.ts:33]: Rename to `Accounts`

File: scripts/generate-accounts/index.ts [hunk 2]
```diff
@@ -23,12 +24,21 @@ class Accounts {
...
+    console.log(`\n`);
+
+    const generateSocietyAccount = await this.generateSocietyAccount();
```
Comment (AndreiNavoichyk) [inline scripts/generate-accounts/index.ts:36]: Rename to `Accounts`

File: scripts/generate-accounts/index.ts [hunk 3]
```diff
@@ -23,12 +24,21 @@ class Accounts {
...
+    console.log(`\n`);
+
+    const generateTechCommAccount = await this.generateTechCommAccount();
```
Comment (AndreiNavoichyk) [inline scripts/generate-accounts/index.ts:39]: Rename to `Accounts`

**Review by AndreiNavoichyk — APPROVED** [review]

File: scripts/generate-accounts/index.ts (hunk 1 above)
Comment (karthikeyan-cere) [reply scripts/generate-accounts/index.ts]: renamed

File: scripts/generate-accounts/index.ts (hunk 2 above)
Comment (karthikeyan-cere) [reply scripts/generate-accounts/index.ts]: renamed

File: scripts/generate-accounts/index.ts (hunk 3 above)
Comment (karthikeyan-cere) [reply scripts/generate-accounts/index.ts]: renamed

**Pull request merged**